src/client/app/redux/actions/meters.ts

```diff
@@ -1,65 +1,41 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
-  * License, v. 2.0. If a copy of the MPL was not distributed with this
-  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+	* License, v. 2.0. If a copy of the MPL was not distributed with this
+	* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { ActionType, Thunk, Dispatch, GetState } from '../types/redux/actions';
-import { showSuccessNotification } from '../utils/notifications';
-import translate from '../utils/translate';
-import * as t from '../types/redux/meters';
-import { metersApi } from '../utils/api';
-import { updateCikAndDBViewsIfNeeded } from './admin';
 
-export function requestMetersDetails(): t.RequestMetersDetailsAction {
-	return { type: ActionType.RequestMetersDetails };
-}
+// TODO Marked For Deletion after RTK migration solidified
+/* eslint-disable jsdoc/check-param-names */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-nocheck
+/* eslint-disable jsdoc/require-param */
+
+import { Dispatch, GetState, Thunk } from '../../types/redux/actions';
+import * as t from '../../types/redux/meters';
+import { metersApi } from '../../utils/api';
+import { showErrorNotification, showSuccessNotification } from '../../utils/notifications';
+import translate from '../../utils/translate';
+import { updateCikAndDBViewsIfNeeded } from './admin';
 
-export function receiveMetersDetails(data: t.MeterData[]): t.ReceiveMetersDetailsAction {
-	return { type: ActionType.ReceiveMetersDetails, data };
-}
 
 export function fetchMetersDetails(): Thunk {
```

Comment:
This function is still in a comment in src/client/app/components/csv/ReadingsCSVUploadComponent.tsx but maybe it can be cleaned up and removed. The rest seem to be unused so can delete per TODO above.

Reply:
done